src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -992,7 +992,7 @@ class VM_PrintClasses : public VM_Operation {
 };
 
 void ClassesDCmd::execute(DCmdSource source, TRAPS) {
-  VM_PrintClasses vmop(output(), _verbose.is_set());
+  VM_PrintClasses vmop(output(), _verbose.value());
```

Comment:
What if it wasn't set?

Reply:
It has a default value...  Generally "false" for _verbose DCmdArguments.

Reply:
Ok

Reply:
Thanks!